android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -235,6 +237,27 @@ public void onError(@NonNull RegistrationException error,
         };
     }
 
+    private UnregistrationListener unregistrationListener() {   
+        return new UnregistrationListener() {
+            @Override
+            public void onUnregistered(String accessToken, String fcmToken) {
+                if (BuildConfig.DEBUG) {
+                    Log.d(TAG, "Successfully unregistered FCM");
+                }
+                // eventManager.sendEvent(EVENT_DEVICE_UNREGISTERED, null);
+                eventManager.sendEvent(EVENT_DEVICE_NOT_READY, null);
+            }
+
+            @Override
+            public void onError(RegistrationException error, String accessToken, String fcmToken) {
+                Log.e(TAG, String.format("Unregistration Error: %d, %s", error.getErrorCode(), error.getMessage()));
+                WritableMap params = Arguments.createMap();
+                params.putString("err", error.getMessage());
```

Comment:
use Constants.ERROR rather than "err" to avoid typo errors

Reply:
done